Formula/yarn.rb

```diff
@@ -18,9 +16,10 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
-    inreplace "#{libexec}/lib/node_modules/yarn/package.json", '"installationMethod": "tar"', '"installationMethod": "homebrew"'
+    libexec.install Dir["*"]
+    bin.install_symlink "#{libexec}/bin/yarn.js" => "yarn"
+    bin.install_symlink "#{libexec}/bin/yarn.js" => "yarnpkg"
```

Comment:
Those are extracted from the `bin` object in package.json. Is there some easy way to use something like `jq` to extract it?

https://github.com/yarnpkg/yarn/blob/76c9fc181e819d76d6f0833f57313c646b9b5406/package.json#L82-L85